src/library/styles/pxToEm.js

```diff
@@ -4,8 +4,20 @@
  */
 import fontSize_base from '../themes/fontSizeBase';
 
+const justNumbersOrEndsWithPx = (s: string) => s.match(/^[\d.]+(px)?$/);
+
+const convert = (input: number): string =>
+  `${input / parseInt(fontSize_base)}em`;
+
+const coerce = (value) => (typeof value === 'string' ? value : typeof value);
+const errorMsg = (actual) =>
+  `[mineral-ui/styles/pxToEm]: Expected a number or string in px units. Instead got: '${coerce(
+    actual
+  )}'.`;
+
 export default function pxToEm(value: number | string) {
-  const input =
-    typeof value === 'string' ? parseInt(value.replace('px', '')) : value;
-  return `${input / parseInt(fontSize_base)}em`;
+  if (typeof value === 'number') return convert(value);
```

Comment:
Not sure why these are not eslint `curly` errors.  We should probably always use braces here. 
 https://eslint.org/docs/rules/curly

Suggest to add `"curly": ["error"]` to `.eslintrc` rules section.